### ISSUE TITLE
SOCKS access for [Lindsay Insco]

### ISSUE THREAD

**Issue by lindsay-insco** [department-of-veterans-affairs/va.gov-team#61638]
### Your Name

Lindsay Insco

### Your Email

lindsay.insco@oddball.io

### Your Role and Team

Platform Support Tier 1 

### Product Manager (PM) name and email

Alyssa Gallion - alyssa.gallion@oddball.io 

### Product Owner (PO) name and email

Erika Washburn - Erika.Washburn@va.gov

### Access Type Requesting

platform

### Public SSH Key

ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDDk+wJLkTgt5tpMgKCf7jGdMadHqp+DBBFVnrpZrlIC6WB4s4nX53FCrXqEZXzop8bxwTFtEiSLXJiwxpr6HBdb3fEXDp1GBRYMPuiZOsjR00lGY7lZigP5tzQzs8aZJzhYeVsJNMjgP4wWOaQDXrreIA8qPrEgdswPtSveQvStg50ypYCHZ5sVX4rnqXWTRvjtjKkwxCA9GAfCD7Q/mHs2EfS1RTQvAVAT9UH0cgFTqbSolKfBC99cTv3uYfhuLEM1ykLCLnoZJeN9Ivsv/NNS2gWGU0MJJQe5nEXenXaj71De0NZPdZHha0Ey/FVTJC8/v27avB2Rwjca3LXitbp5R4lXQDhvQ+swCugrqu8I6UhkpnKTmo8qDVgmK9myqph+OO4GKcwc/nRjVNMDo8XEIAUAdxnqBVYffN6pWj033WcNOg2KX2WIGaV296ko40TLYtVr1VtW2eP81p87UqVeb0OnPV+TvMp9P0/Uh5cih2DirZqlOwolHBvNyHWcF0= lindsayinsco@Lindsays-MacBook-Air.local

### E-QIP Transmittal Confirmation

![Screenshot 2023-07-07 at 5 52 00 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/90409405/d835bee1-2b78-475d-a749-d5ce8f77cbca)

### User must exist in a roster before SOCKS access can be granted

- [ ] Search for user on the VFS Team Roster: https://docs.google.com/spreadsheets/d/11dpCJjhs007uC6CWJI6djy3OAvjB8rHB65m0Yj8HXIw/edit?folder=0ALlyxurHpUilUk9PVA#gid=2042046665
- [ ] Or search for user on the Platform Team Roster: https://vfs.atlassian.net/wiki/spaces/AP/pages/1908834623/Platform+Roster
- [ ] If user is on a VFS team but not in the VFS Team Roster, add the 'NOT YET' label and instruct them to start the Platform orientation process https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html
- [ ] If a user is on a Platform team but not on the Platform Team Roster in Confluence, add the 'NOT YET' label and instruct them to reach out to their Product Manager to be added.
- [ ] Comment in this issue saying which roster the user is listed in.

**Comment by rmtolmach** [comment]
hey @lindsay-insco! We found out last week (when you were out, I think) that our PO is Erika Washburn, so I edited the description to reflect that. 

Also... 😕 not sure why you're on the VFS roster. And listed under the Modernized check-in team?? Is that a mistake?
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/10993987/3ca7f038-9008-4696-bff2-3add87e54597)

But you ARE also on the Platform roster, so we're good ✅
